src/runtime/security/meta_access_controller.cpp

```diff
@@ -0,0 +1,68 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "meta_access_controller.h"
+
+#include <dsn/tool-api/rpc_message.h>
+#include <dsn/utility/flags.h>
+#include <dsn/tool-api/network.h>
+#include <dsn/dist/fmt_logging.h>
+
+namespace dsn {
+namespace security {
+DSN_DEFINE_string("security",
+                  rpc_code_allowed_list,
```

Comment:
```suggestion
                  meta_acl_rpc_allow_list,
```

I can't read from the config name what it's used for. Rename it with the keyword "meta" and "acl".